server/platform/services/searchengine/bleveengine/indexer/indexing_job.go

```diff
@@ -472,7 +472,9 @@ func (worker *BleveIndexerWorker) BulkIndexFiles(files []*model.FileForIndexing,
 	batch := worker.engine.FileIndex.NewBatch()
 
 	for _, file := range files {
-		if file.DeleteAt == 0 {
+		shouldIndexFile := file.DeleteAt == 0 && (file.PostId != "" || file.CreatorId == model.BookmarkFileOwner)
```

Comment:
Pending my comment in the other related PR

Reply:
Fixed.